test/controllers/returns/credits/RemoveCreditControllerSpec.scala

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.returns.credits
+
+import base.SpecBase
+import connectors.CacheConnector
+import forms.returns.credits.RemoveCreditFormProvider
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.inject.bind
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import views.html.returns.credits.RemoveCreditView
+
+import scala.concurrent.Future
+
+class RemoveCreditControllerSpec extends SpecBase with MockitoSugar {
+
+  val formProvider = new RemoveCreditFormProvider()
+  val form = formProvider()
+
+  lazy val removeCreditRoute = controllers.returns.credits.routes.RemoveCreditController.onPageLoad().url
+
+  "RemoveCredit Controller" - {
+
+    "must return OK and the correct view for a GET" in {
+
+      val application = applicationBuilder(userAnswers = Some(emptyUserAnswers)).build()
+
+      running(application) {
```

Comment:
I thought we wanted to get away from loading the application in unit test. Anyway not a blocker as this still valid tests.

Reply:
We do,but this is just what it generated when i used scaffold.